lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -2337,13 +2337,13 @@ namespace Js
             return si;
     }
 
-#ifdef ENABLE_GLOBALIZATION
     void ScriptContext::UpdateTimeZoneInfo()
     {
         GetTimeZoneInformation(&timeZoneInfo);
+#if !defined(__cplusplus_winrt) && defined(_WIN32)
```

Comment:
> I'm unfamiliar with it. It might be possible that chakra.dll can use this API? Maybe better avoid side-fixing it here and file an issue to investigate it separately?

Okay, added a comment there instead.